app/javascript/article-form/components/ArticleCoverImage.jsx

```diff
@@ -91,7 +91,10 @@ export class ArticleCoverImage extends Component {
   };
 
   useNativeUpload = () => {
-    return Runtime.isNativeIOS('imageUpload');
+    // This namespace is not implemented in the native side. This allows us to
+    // deploy our refactor and wait until our iOS app is approved by AppStore
+    // review. The old web implementation will be the fallback until then.
+    return Runtime.isNativeIOS('imageUpload_disabled');
```

Comment:
The context lives right here in the inline comments, but here's more context on this decision: https://forem-team.slack.com/archives/C02J81R0LKE/p1637700468071300?thread_ts=1637695134.070100&cid=C02J81R0LKE